### PR TITLE
[etcd] Update plugin for containerized deployment

### PR DESCRIPTION
**Pull request by TurboTurtle**
Today, etcd is almost exclusively deployed in a container for kubernetes/OCP clusters. This commit serves to update the plugin to be primarily designed for those types of deployments, and removes the v1 and v2 collections.

Resolves: #2906

Signed-off-by: Jake Hunsaker <jhunsake@redhat.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [x] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

**Review by pmoravec — APPROVED** [review]

File: sos/report/plugins/etcd.py [hunk 1]
```diff
@@ -13,71 +13,54 @@
...
+    """
 
     short_desc = 'etcd plugin'
 
     plugin_name = 'etcd'
     packages = ('etcd',)
     profiles = ('container', 'system', 'services', 'cluster')
-    files = ('/etc/origin/node/pods/etcd.yaml',)
+    files = ('/etc/etcd/etcd.conf',)
+    containers = ('etcdctl',)
```
[the remark this answers is not in the document]
Comment (TurboTurtle) [reply sos/report/plugins/etcd.py:34]: What environment is that? Testing on OCP has the `etcdctl` container within the `etcd` pod, with the container being intended for etcdctl execution.

[the remark this answers is not in the document]
Comment (vikas-goel) [reply sos/report/plugins/etcd.py:34]: That’s Veritas Appliances. Can the plug-in take a list of container names for that matter as a plug-in option?

[the remark this answers is not in the document]
Comment (TurboTurtle) [reply sos/report/plugins/etcd.py:34]: I'm hesitant to put it as a plugin option, however we can certainly add the stock `etcd` name to the `containers` tuple and iterate over that looking for a matching container

[the remark this answers is not in the document]
Comment (vikas-goel) [reply sos/report/plugins/etcd.py:34]: Sounds good

**Comment by TurboTurtle** [comment]
@vikas-goel I pushed an update to try and account for the container name being `etcd`. PTAL.

**Pull request merged**